syscall/devices/hostepoll/hostepoll.c

```diff
@@ -362,7 +362,7 @@ static int _epoll_ctl_del(epoll_t* epoll, int fd)
     {
         bool found = false;
 
-        for (size_t i = 0; epoll->map_size; i++)
+        for (size_t i = 0; i < epoll->map_size; i++)
```

Comment:
@mingweishih Is there some compiler flag that can detect such coding errors?

Reply:
I don't this can be detected with compiler flags as this is legitimate in syntax-wise. However, some static analysis tools should help (maybe Clang already supports some).